src/enterdb_lib.erl

```diff
@@ -1779,6 +1779,15 @@ get_indexed_cids(_Mapper, [], Acc) ->
 get_index_terms(Mapper, IndexOn, Columns) ->
     get_index_terms(Mapper, IndexOn, Columns, []).
 
+get_index_terms(Mapper, [{Col, erl_term} | Rest], Columns, Acc) ->
+    case lists:keyfind(Col, 1, Columns) of
+	{_, Value} ->
+	    Cid = encode_unsigned(2, Mapper:lookup(Col)),
+	    BinTerms = encode_erlang_term(Value),
```

Comment:
This function could be still make_index_terms/2 but a function_clause could handle 'erl_term'. Same for string_to_binary_terms/1., or it could be called from make_index_terms/2.